test/suite/unit/Exception/FrozenAttributeException.spec.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use Rinq\Ident\PeerId;
+use Rinq\Ident\Reference;
+use Rinq\Ident\SessionId;
+
+describe('FrozenAttributeException', function () {
```

Comment:
I like to do `FrozenAttributeException::class` for the outer description. Your call, it just helps when you end up with the same class name used in different namespaces.